hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CapacityScheduler.java

```diff
@@ -3364,12 +3364,29 @@ public boolean moveReservedContainer(RMContainer toBeMovedContainer,
     }
   }
 
-  @Override
+   @Override
   public long checkAndGetApplicationLifetime(String queueName,
       long lifetimeRequestedByApp) {
+    CSQueue queue = getQueue(queueName);
+
+    // This handles the case where the queue does not exist,
+    // addressing the issue related to YARN-11708.
+    if (queue == null) {
+      QueuePath queuePath = new QueuePath(queueName);
+
+      writeLock.lock();
+      try {
+        queue = queueManager.createQueue(queuePath);
+      } catch (YarnException | IOException e) {
+        LOG.error("Failed to create queue '{}': ", queueName, e);
```

Comment:
```suggestion
        LOG.error("Failed to create queue " + queueName, e);
```

Because the current log line wont log the exception, cause will think that is the 2nd param in the message